### PR TITLE
New version: ObinsKit.ObinsKit version 1.2.10

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38214)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ObinsKit.ObinsKit
```
Comment (palenshus) [inline manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml:4]: Why is the Publisher portion of the PackageId Obinskit? Shouldn't it be Hexcore, or whatever else the Publisher reports as

Comment (OfficialEsco) [reply manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml:4]: blame wingetcreate https://github.com/microsoft/winget-pkgs/pull/17267 🤷‍♂️ Also a bunch of manifests have the PackageName as Publisher in the PackageID (Mostly from Manifestversion 0.0.1)

Where are you getting Hexcore from? Would you prefer `TaicangZhigengniaoInformationTechnology.ObinsKit` instead?

Comment (palenshus) [reply manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml:4]: I definitely like HexCore better than that :D I got it from the domain, and their about page: https://www.hexcore.xyz/about: 

> Hexcore is a brand founded by a group of designers and programmers full of dreams, dedicated to become an excellent peripherals company.

That said, I don't feel super strongly about it if you'd rather not change it

Comment (OfficialEsco) [reply manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml:4]: Well, this is a new version PR, so we would have to move the Package.

Comment (palenshus) [reply manifests/o/ObinsKit/ObinsKit/1.2.10/ObinsKit.ObinsKit.installer.yaml:4]: Yeah, perhaps not worth doing, feel free to resolve

**Pull request merged**